Apps/AdminWebClient/src/ClientApp/src/App.vue

```diff
@@ -2,9 +2,9 @@
     <v-app id="healthGatewayAdmin">
         <ToolbarComponent />
         <MainNavbar title="HealthGateway Admin" />
-        <v-main>
+        <v-content>
```

Comment:
can you explain why this?

Reply:
Those tags where deprecated and where showing warnings. v-content is the new tag